vite-project-kabinet/src/Content/Content.jsx

```diff
@@ -1,16 +1,18 @@
 import "./content.style.css"
-import { Card } from "./Card.jsx"
+import { PaginatedList } from "./PaginatedList.jsx"
+import { LoadedList } from "./LoadedList.jsx"
 
 
 export const Content = ({articles}) => {
 
   return (
-        <main>
-            <section className="card__container">
-              {articles.map(article =>
-                <Card key={article.id} genre={article.genre} title={article.title} author={article.author} content={article.content} image={article.image} />
-              )}
-            </section>
-        </main>
+    <main>
+      <div className="mobile-only">
+          <LoadedList items={articles} itemsPerPage={4}/>
+      </div>
+      <div className="desktop-only">
+          <PaginatedList items={articles} itemsPerPage={6}/>
+      </div>
```

Comment:
React Ti tady vykreslí články 2x a pak je CSS jednou schová. Lepší je zjistit si šířku viewportu přímo v Reactu a pak vypsat jen tu část, která je relevantní. 

Jsou na to knihovny (já kdysi používal `useBreakpoints`), nebo si to můžeš zkusit napsat sama. Ale budeš muset ošetřit situaci, kdy velikost viewportu se Ti může kdykoli změnit a je třeba na to reagovat a překreslit stránku.

Reply:
Upravila som komponentu Content tak, aby sa relevantný obsah zobrazoval podľa šírky viewportu. Pushla som to do feature/routing.  Bolo správne použiť useEffect , ktorý sa spúšťa len prvý krát pri načítaní komponenty? Trochu som sa tento víkend venovala useEffect v Reactu a rozumiem dovodom, prečo je doležitý napr. pri fetch, ale v tomto prípade mi účel useEffectu trochu uniká...